draft-ietf-quic-recovery.md

```diff
@@ -1265,7 +1244,8 @@ OnLossDetectionTimeout():
     GetEarliestTimeAndSpace(loss_time)
   if (earliest_loss_time != 0):
     // Time threshold loss Detection
-    DetectLostPackets(pn_space)
+    lost_packets = DetectLostPackets(pn_space)
+    OnPacketsLost(lost_packets)
```

Comment:
```suggestion
    if (!lost_packets.empty()):
      OnPacketsLost(lost_packets)
```

Reply:
If a timeout occurred, then lost_packets should never be empty I believe?  I could add an assert for that if you'd like?

Reply:
Hm, that's true. I was thinking about stream cancellations, but that doesn't matter here. assert SGTM